pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/QueryRunner.java

```diff
@@ -142,9 +144,10 @@ public void processQuery(DistributedStagePlan distributedStagePlan, Map<String,
             new ServerMetrics(PinotMetricUtils.getPinotMetricsRegistry()), System.currentTimeMillis());
         serverQueryResults.add(processServerQuery(request, _scheduler.getWorkerPool()));
       }
-      LOGGER.debug(
-          "RequestId:" + requestId + " StageId:" + distributedStagePlan.getStageId() + " Leaf stage v1 processing time:"
-              + (System.currentTimeMillis() - leafStageStartMillis) + " ms");
+      if (shouldLogOpStats) {
+        LOGGER.error("RequestId:" + requestId + " StageId:" + distributedStagePlan.getStageId()
```

Comment:
error seems like a weird log level for this - why not make it INFO and then just have people configure log4j properly for it?

Reply:
+1 lets put this in info. we can remove this once join is GA

Reply:
Because it is easier to see in IDE..... the whole stats is quite hacky anyway. So I put loggeer.error.
We probably need to rewrite the logging later.

Reply:
updated to log info